datafusion/functions/src/datetime/to_date.rs

```diff
@@ -162,7 +151,7 @@ impl ScalarUDFImpl for ToDateFunc {
     }
 
     fn documentation(&self) -> Option<&Documentation> {
```

Comment:
I dont have better solutions here for now, so every impl have to copy this line because its not easy to inject/override a method into trait impl, I'll experiment more with this, but for now it is what it is

Reply:
I think as long as it is well documented, this boiler plate is ok

Reply:
Maybe we could call the function something like `macro_doc` or `derived_documentation` to make it easier to find (aka so someone who runs across it can easily `grep` and find the macro definition)